crates/bevy_ecs/src/system/commands/parallel_scope.rs

```diff
@@ -14,7 +14,8 @@ struct ParallelCommandQueue {
     thread_queues: Parallel<CommandQueue>,
 }
 
-/// An alternative to [`Commands`] that can be used in parallel contexts, such as those in [`Query::par_iter`](crate::system::Query::par_iter)
+/// An alternative to [`Commands`] that can be used in parallel contexts, such as those
+/// in [`Query::par_iter`](crate::system::Query::par_iter).
```

Comment:
We should add a quick note to these docs pointing users at `spawn_batch` here still :)

Reply:
Should I link to the `World::spawn_batch` or the `Commands::spawn_batch`?

Reply:
we're in commands so the latter